package.json

```diff
@@ -58,6 +58,8 @@
     "sass-loader": "7.2.0",
     "semver": "6.3.0",
     "shards-react": "^1.0.3",
+    "sinon": "^7.5.0",
+    "sinon-stub-promise": "^4.0.0",
```

Comment:
to prevent further delays in our progress I like this move. 

I approve of this